### PR TITLE
{Release} Upgrade to Azure CLI 2.11.1

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,27 @@
...
+2.11.1
+++++++
+
+**ACR**
+
+* Add Isolated Tier to Agent Pool (#14952)
+
+**AKS**
+
+* fix: aks cluster create issue (#14920)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:15]:
```suggestion
* Fix aks cluster create issue (#14920)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
+* Fix aks cluster create issue (#14920)
+
+**Cognitive Services**
+
+* [BREAKING CHANGE] Show additional legal term for certain APIs (#14901)
+
+**Network**
+
+* [BREAKING CHANGE] Allow to create both public and private IP while creating an Application Gateway (#14874)
+* Fix help message of options about connection-monitor endpoint (#14922)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:24]:
```suggestion
```

Comment (MyronFanQiu) [inline src/azure-cli/HISTORY.rst:24]:
```suggestion

```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,26 @@
...
+
+**Cognitive Services**
+
+* [BREAKING CHANGE] Show additional legal term for certain APIs (#14901)
+
+**Network**
+
+* [BREAKING CHANGE] Allow to create both public and private IP while creating an Application Gateway (#14874)
+* `az network list-service-tags`: add details on location parameter use to the help message (#14935)
+
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:25]:
```suggestion
**Storage**

 * `az storage blob list`: Support OR properties with new api version
 
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,29 @@
...
+**Cognitive Services**
+
+* [BREAKING CHANGE] Show additional legal term for certain APIs (#14901)
+
+**Network**
+
+* [BREAKING CHANGE] Allow to create both public and private IP while creating an Application Gateway (#14874)
+* `az network list-service-tags`: add details on location parameter use to the help message (#14935)
+**Storage**
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:25]:
```suggestion

**Storage**
```

**Review by fengzhou-msft — APPROVED** [review]

**Review by arrownj — APPROVED** [review]
LGTM

**Comment by yonzhan** [comment]
Release

**Pull request merged**